core/linux-dist/x11.cpp

```diff
@@ -107,8 +125,9 @@ void input_x11_handle()
 						{
 							kcode[0] |= dc_key;
 						}
+
+						printf("KEY: %d -> %d: %d\n", e.xkey.keycode, dc_key, x11_dc_buttons );
```

Comment:
I'd comment out this one, or make it use some debug log and not printf to avoid console spam

Reply:
Should I use ``#if defined(_DEBUG)``?